src/AbstractFixer.php

```diff
@@ -45,6 +46,13 @@ public function __construct()
         }
     }
 
+    public function fix(\SplFileInfo $file, Tokens $tokens)
```

Comment:
Since now Fixers are obliged to implement `applyFix` method, it would be better to make this method final, so Fixer creators are not confused implementing both.
It would be a BC break, but maybe on 2.3 would be ok